cli/cmd/admin/start.go

```diff
@@ -37,47 +39,51 @@ import (
-	ActivitySinkType         string                 `default:"" split_words:"true"`
-	ActivitySinkKafkaBrokers string                 `default:"" split_words:"true"`
-	ActivityUISinkKafkaTopic string                 `default:"" split_words:"true"`
-	MetricsProject           string                 `default:"" split_words:"true"`
-	AutoscalerCron           string                 `default:"CRON_TZ=America/Los_Angeles 0 0 * * 1" split_words:"true"`
+	DatabaseDriver         string                 `default:"postgres" split_words:"true"`
+	DatabaseURL            string                 `split_words:"true"`
+	RedisURL               string                 `default:"" split_words:"true"`
+	ProvisionerSetJSON     string                 `split_words:"true"`
+	DefaultProvisioner     string                 `split_words:"true"`
+	Jobs                   []string               `split_words:"true"`
+	LogLevel               zapcore.Level          `default:"info" split_words:"true"`
+	MetricsExporter        observability.Exporter `default:"prometheus" split_words:"true"`
+	TracesExporter         observability.Exporter `default:"" split_words:"true"`
+	HTTPPort               int                    `default:"8080" split_words:"true"`
+	GRPCPort               int                    `default:"9090" split_words:"true"`
+	DebugPort              int                    `split_words:"true"`
+	ExternalURL            string                 `default:"http://localhost:8080" split_words:"true"`
+	ExternalGRPCURL        string                 `envconfig:"external_grpc_url"`
+	FrontendURL            string                 `default:"http://localhost:3000" split_words:"true"`
+	AllowedOrigins         []string               `default:"*" split_words:"true"`
+	SessionKeyPairs        []string               `split_words:"true"`
+	SigningJWKS            string                 `split_words:"true"`
+	SigningKeyID           string                 `split_words:"true"`
+	AuthDomain             string                 `split_words:"true"`
+	AuthClientID           string                 `split_words:"true"`
+	AuthClientSecret       string                 `split_words:"true"`
+	GithubAppID            int64                  `split_words:"true"`
+	GithubAppName          string                 `split_words:"true"`
+	GithubAppPrivateKey    string                 `split_words:"true"`
+	GithubAppWebhookSecret string                 `split_words:"true"`
+	GithubClientID         string                 `split_words:"true"`
+	GithubClientSecret     string                 `split_words:"true"`
+	AssetsBucket           string                 `split_words:"true"`
+	// AssetsBucketGoogleCredentialsJSON is only required to be set for local development.
+	// For production use cases the service account will be directly attached to pods which is the recommended way of setting credentials.
+	AssetsBucketGoogleCredentialsJSON string `split_words:"true"`
```

Comment:
Did you also update the dev `.env` file with values for these? (run `rill devtool dotenv upload cloud`)

Reply:
Yes I update the `.env` file